meta_request/lib/meta_request/app_notifications.rb

```diff
@@ -81,6 +81,11 @@ def subscribe(event_name)
       self
     end
 
+    private
+
+    def self.is_dev_caller(file_path)
```

Comment:
How about `dev_caller?`

Reply:
Good point. That would be more idiomatic Ruby. Done.